version

```diff
@@ -0,0 +1,4 @@
+package fizz
+
+// Version of fizz
+const Version = "v0.0.1"
```

Comment:
This file seems weird. :)

Reply:
I rewrote the release stuff. It’ll get updated when a release is cut. Don’t worry too much about it.